packages/core/utils/src/totals/promotion/index.ts

```diff
@@ -5,23 +5,23 @@ import {
 } from "../../promotion"
 import { MathBN } from "../math"
 
-function getPromotionValueForPercentage(promotion, lineItemTotal) {
-  return MathBN.mult(MathBN.div(promotion.value, 100), lineItemTotal)
+function getPromotionValueForPercentage(sumTaxRate, promotion, lineItemAmount) {
```

Comment:
todo: I believe we can remove the `sumTaxRate`; it's no longer used
```suggestion
function getPromotionValueForPercentage(promotion, lineItemAmount) {
```